### ISSUE TITLE
БМЦИС-F4 выбор УПО

### ISSUE THREAD

**Issue by CrIcKeT98** [metrostroi-repo/MetrostroiAddon#12]

**Comment by kosmik641** [comment]
Добавить типа галки на F4?

**Comment by glebqip** [comment]
Да, вообще интерфейс БМЦИС надо привести к современному варианту, тогда отпадёт необходимость в автомате УПО1/УПО2

**Comment by kosmik641** [comment]
Нутут я вижу 2 варианта:
1. Впихнуть то, что уже есть (https://steamcommunity.com/sharedfiles/filedetails/?id=1917660349)
2. Тупо текущий БМЦИС привести к современному виду без изменения логики

**Comment by glebqip** [comment]
Ну БМЦИС можно привести к реальному варианту вообще. Первый вариант да, код весь надо смотреть.
